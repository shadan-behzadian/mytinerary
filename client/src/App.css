.HomeImagesCenter {
  display: flex;
  justify-content: center;
}

.logo {
  margin-top: 5%;

  height: 10%;
  width: 40%;
}

.arrow {
  height: 10%;
  width: 10%;
}

.homePagetxt {
  text-align: center;
}

.father {
  display: flex;
  justify-content: center;
}

.myChild {
  flex-direction: row;
  padding: 10px;
}

.Home {
  position: fixed;
  left: 0;
  bottom: 0;
  width: 100%;
  background-color: gray;
  text-align: center;
  font-size: 30px;
}

.myHome {
  width: 50px;
  height: 50px;
}

header {
  left: 0;
  bottom: 0;
  width: 100%;
  background-color: gray;
  text-align: right;
  font-size: 30px;
  display: flex;
  flex-direction: row;
}

/* .SideBar {
  font-size: 30px;
  padding: 10px;
  display: inline;
} */

.UserLogIn {
  font-size: 20px;
  padding: 25px;
  display: inline;
}

/* Position and sizing of burger button */
.bm-burger-button {
  position: fixed;
  width: 36px;
  height: 30px;
  right: 20px;
  top: 20px;
}

/* Color/shape of burger icon bars */
.bm-burger-bars {
  background: #373a47;
}

/* Position and sizing of clickable cross button */
.bm-cross-button {
  height: 24px;
  width: 24px;
}

/* Color/shape of close button cross */
.bm-cross {
  background: #bdc3c7;
}

/* General sidebar styles */
.bm-menu {
  background: #373a47;
  padding: 2.5em 1.5em 0;
  font-size: 1.15em;
}

/* Morph shape necessary with bubble or elastic */
.bm-morph-shape {
  fill: #373a47;
}

/* Wrapper for item list */
.bm-item-list {
  color: #b8b7ad;
  padding: 0.8em;
}

/* Individual item */
.bm-item {
  display: inline-block;
}

/* Styling of overlay */
.bm-overlay {
  background: rgba(0, 0, 0, 0.3);
}

.AllCities {
  font-size: 100%;
  text-align: center;
}

.cityPage {
  width: 50%;
  display: inline-block;
  margin: auto;
  border-style: groove;
}

.cityPagemain {
  width: 50%;
  margin: auto;
  border-style: groove;
}

.removeBtn {
  display: inline-block;
}
.profilePic {
  width: 8%;
  height: 8%;
  border-radius: 100%;
}

.cityAdmin {
  font-size: 100%;
  text-align: center;
}

.removeCityPage {
  margin: auto;
}

/* .removeCity {
  width: 50%;
  border-style: groove;
} */

.disabled-link {
  pointer-events: none;
}

.EachMytinerary {
  display: flex;
  margin: auto;
  width: 50%;
  border-style: dotted;
  margin-bottom: 10px;
}

.profileNavBar {
  width: 100%;
  height: 300px;
}

.profileButton {
  width: 27%;
  height: 60%;
  border-style: outset;
  font-size: x-large;
  border-radius: 33px;
}

.space {
  margin-right: 10px;
}

.mytineraryForm {
  display: flex;
  flex-direction: column;
}

.mytineraryForm input {
  width: 40%;
  margin: auto;
  margin-bottom: 5px;
}
