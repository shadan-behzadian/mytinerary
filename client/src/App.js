import React, { Component } from "react";

import "./App.css";
import { BrowserRouter } from "react-router-dom";

import Header from "./components/Header";

import Footer from "./components/Footer";
import Router from "./components/Router";
class App extends Component {
  render() {
    return (
      <BrowserRouter>
        <div className="App">
          <Header />
          <Router />
          <Footer />
          <div>hi</div>
        </div>
      </BrowserRouter>
    );
  }
}

export default App;
