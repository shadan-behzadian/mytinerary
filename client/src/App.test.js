import React from "react";
import ReactDOM from "react-dom";
import App from "./App.js";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

//with shallow
import { shallow } from "enzyme";

it("shallow renders without crashing", () => {
  shallow(<App />);
});

it("renders correctly", () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
