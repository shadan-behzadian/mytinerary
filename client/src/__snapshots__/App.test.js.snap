// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <div
    className="Header"
  >
    <a
      href="/profile"
      onClick={[Function]}
    >
      <div
        className="UserLogIn"
      >
        <button>
          <i
            className="glyphicon glyphicon-user"
          />
        </button>
      </div>
    </a>
    <div>
      <div
        className="bm-overlay"
        onClick={[Function]}
        style={
          Object {
            "MozTransform": "translate3d(100%, 0, 0)",
            "MsTransform": "translate3d(100%, 0, 0)",
            "OTransform": "translate3d(100%, 0, 0)",
            "WebkitTransform": "translate3d(100%, 0, 0)",
            "background": "rgba(0, 0, 0, 0.3)",
            "height": "100%",
            "opacity": 0,
            "position": "fixed",
            "transform": "translate3d(100%, 0, 0)",
            "transition": "opacity 0.3s, transform 0s 0.3s",
            "width": "100%",
            "zIndex": 1000,
          }
        }
      />
      <div
        className="bm-menu-wrap"
        id=""
        style={
          Object {
            "MozTransform": "translate3d(100%, 0, 0)",
            "MsTransform": "translate3d(100%, 0, 0)",
            "OTransform": "translate3d(100%, 0, 0)",
            "WebkitTransform": "translate3d(100%, 0, 0)",
            "height": "100%",
            "position": "fixed",
            "right": 0,
            "transform": "translate3d(100%, 0, 0)",
            "transition": "all 0.5s",
            "width": "15%",
            "zIndex": 1100,
          }
        }
      >
        <div
          className="bm-menu"
          style={
            Object {
              "boxSizing": "border-box",
              "height": "100%",
              "overflow": "auto",
            }
          }
        >
          <nav
            className="bm-item-list"
            style={
              Object {
                "height": "100%",
              }
            }
          >
            <a
              aria-current={null}
              className="bm-item menu-item"
              href="/LogIn"
              id="home"
              onClick={[Function]}
              style={
                Object {
                  "display": "block",
                }
              }
              tabIndex={-1}
            >
              LogIn
            </a>
            <a
              aria-current={null}
              className="bm-item menu-item"
              href="/SignUp"
              id="about"
              onClick={[Function]}
              style={
                Object {
                  "display": "block",
                }
              }
              tabIndex={-1}
            >
              Create Account
            </a>
          </nav>
        </div>
        <div
          style={Object {}}
        >
          <div
            className="bm-cross-button"
            style={
              Object {
                "height": 24,
                "position": "absolute",
                "right": 8,
                "top": 8,
                "width": 24,
              }
            }
          >
            <span
              style={
                Object {
                  "position": "absolute",
                  "right": "14px",
                  "top": "6px",
                }
              }
            >
              <span
                className="bm-cross"
                style={
                  Object {
                    "height": 14,
                    "position": "absolute",
                    "transform": "rotate(45deg)",
                    "width": 3,
                  }
                }
              />
              <span
                className="bm-cross"
                style={
                  Object {
                    "height": 14,
                    "position": "absolute",
                    "transform": "rotate(-45deg)",
                    "width": 3,
                  }
                }
              />
            </span>
            <button
              onClick={[Function]}
              style={
                Object {
                  "background": "transparent",
                  "border": "none",
                  "cursor": "pointer",
                  "fontSize": 0,
                  "height": "100%",
                  "left": 0,
                  "margin": 0,
                  "padding": 0,
                  "position": "absolute",
                  "top": 0,
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              Close Menu
            </button>
          </div>
        </div>
      </div>
      <div
        style={Object {}}
      >
        <div
          className="bm-burger-button"
          style={
            Object {
              "zIndex": 1000,
            }
          }
        >
          <span>
            <span
              className="bm-burger-bars"
              style={
                Object {
                  "height": "20%",
                  "left": 0,
                  "opacity": 1,
                  "position": "absolute",
                  "right": 0,
                  "top": "0%",
                }
              }
            />
            <span
              className="bm-burger-bars"
              style={
                Object {
                  "height": "20%",
                  "left": 0,
                  "opacity": 1,
                  "position": "absolute",
                  "right": 0,
                  "top": "40%",
                }
              }
            />
            <span
              className="bm-burger-bars"
              style={
                Object {
                  "height": "20%",
                  "left": 0,
                  "opacity": 1,
                  "position": "absolute",
                  "right": 0,
                  "top": "80%",
                }
              }
            />
          </span>
          <button
            onClick={[Function]}
            onMouseOut={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "background": "transparent",
                "border": "none",
                "cursor": "pointer",
                "fontSize": 0,
                "height": "100%",
                "left": 0,
                "margin": 0,
                "padding": 0,
                "position": "absolute",
                "top": 0,
                "width": "100%",
              }
            }
          >
            Open Menu
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="LandingPage"
  >
    <div
      className="HomeImagesCenter"
    >
      <img
        alt="MytineraryLogo"
        className="logo"
        src="logo.png"
      />
    </div>
    <div>
      <h3
        className="homePagetxt"
      >
        Find you perfect trip, designed by insiders who know and love their cities
      </h3>
      <h3
        className="homePagetxt"
      >
        Start by Browsing
      </h3>
      <a
        aria-current={null}
        href="/AllCities"
        onClick={[Function]}
      >
        <div
          className="HomeImagesCenter"
        >
          <img
            alt="arrowLogo"
            className="arrow"
            src="arrow.png"
          />
        </div>
      </a>
      <h3
        className="homePagetxt"
      >
         Want to build your own Mytinerary?
      </h3>
    </div>
  </div>
  <div
    className="Footer"
  >
    <div
      className="Home"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        <span
          className="glyphicon glyphicon-home "
        />
      </a>
    </div>
  </div>
  <div>
    hi
  </div>
</div>
`;
